src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -57,6 +60,21 @@ class BankAccountStep extends React.Component {
             routingNumber: props.achData.routingNumber || '',
             accountNumber: props.achData.accountNumber || '',
         };
+
+        this.errorTranslationKeys = {
+            routingNumber: 'bankAccount.error.routingNumber',
+            accountNumber: 'bankAccount.error.accountNumber',
+        };
+    }
+
+    getErrors() {
```

Comment:
We are returning an object here so we should add a JSDoc to show the `@returns {Object}`